### ISSUE TITLE
Error: /lib64/libm.so.6: version `GLIBC_2.27' not found

### ISSUE THREAD

**Issue by Sakaki-Ya** [Brooooooklyn/canvas#480]
When deployed and executed on Vercel, the following error was displayed.

```
2022-06-20T22:55:24.677Z	6dc0167d-4230-4c8e-94c4-f08ce2ba6b8a	ERROR	Error: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /var/task/node_modules/@napi-rs/canvas-linux-x64-gnu/skia.linux-x64-gnu.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1144:18)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/var/task/node_modules/@napi-rs/canvas/js-binding.js:138:31)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12) {
  code: 'ERR_DLOPEN_FAILED'
}
RequestId: 6dc0167d-4230-4c8e-94c4-f08ce2ba6b8a Error: Runtime exited with error: exit status 1
Runtime.ExitError
```
Version 0.1.22

It worked on the local server.  
Also, v0.1.21 will work on Vercel.

**Comment by diogoviannaaraujo** [comment]
Seeing the same error on vercel.
Switching to 0.1.21 did not fix.

**Comment by Brooooooklyn** [comment]
@diogoviannaaraujo fixed in 0.1.23
